include/exadg/grid/mesh_movement_functions.h

```diff
@@ -297,6 +299,48 @@ class RectangleMeshMovementFunctions : public dealii::Function<dim>
   double const                time_period;
 };
 
+
+template<int dim>
+struct RotationMeshMovementData
+{
+  RotationMeshMovementData() : counter_clockwise_degrees_per_s(0.0)
+  {
+  }
+
+  double counter_clockwise_degrees_per_s;
+};
+
+template<int dim>
```

Comment:
As a minimal documentation, we should briefly describe the motion like "rotation around positive z/x3-axis" (in the current code version). As written above, it would make sense to extend the functionality towards rotation around any of the coordinate axis.